src/control/control.cpp

```diff
@@ -154,6 +182,23 @@ QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(
     return nullptr;
 }
 
+//static
+QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getDefaultControl() {
+    auto defaultCO = QSharedPointer<ControlDoublePrivate>(s_defaultCO);
```

Comment:
I consider `s_defaultCO.lock()` more idiomatic than the explicit conversion using the constructor. This would also to omit the redundant type declaration that is only required due to our non-progressive code style rules.